openfisca_uk_data/datasets/frs/frs_enhanced/ncfs_emissions_2019.csv

```diff
@@ -0,0 +1,33 @@
+,category,code_start,carbon_tonnes
```

Comment:
Could we add a `README.md` file to this folder describing this dataset, e.g. linking the source?

Reply:
Added